cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -163,6 +153,7 @@ public static CruiseControl fromCrd(Kafka kafkaAssembly, KafkaVersion.Lookup ver
             cruiseControl.setTlsSidecar(tlsSidecar);
 
             cruiseControl = updateConfiguration(spec, cruiseControl);
+            cruiseControl.kafkaClusterConfig = kafkaSpec.getKafka().getConfig().toString().replace("=", ":");
```

Comment:
what's this `kafkaClusterConfig` the set to the `ENV_VAR_KAFKA_CLUSTER_CONFIG_VALUES` env var? It seems to be used for generating the CC config file.

Reply:
Its used to generate the Kafka configuration file Cruise Control uses for CC status/alerting. Unfortunately, I can't just reference the official Kafka configuration files we use for Kafka. Right now, CC only uses the generated file for the `min.insync.replicas` option from the Kafka configuration. I figured we could pass the whole Kafka configuration here in case more Kafka options are used in the future

Reply:
Are you sure the `min.insync.replicas` is there in the final CC config? You are getting the `Kafka.spec.kafka.config` which contains just the user Kafka configuration so the `min.insync.replicas` field could not be there potentially (but the default value is used).

Reply:
Update for just `min.insync.replicas` setting